docs/weightonlyquant.md

```diff
@@ -135,4 +142,83 @@ loaded_model = AutoModelForCausalLM.from_pretrained(saved_dir)
 |       LLM Runtime (use_llm_runtime=True)      |  &#10004;  |  &#10004;  |
 |       PyTorch (use_llm_runtime=False)      |  stay tuned  | &#10004; |
 
-> Note: Only supports CPU device for now. For LLM runtime model loading usage, please refer to [graph readme](../intel_extension_for_transformers/llm/runtime/graph/README.md#2-run-llm-with-transformer-based-api)
+> Note: For LLM runtime model loading usage, please refer to [graph readme](../intel_extension_for_transformers/llm/runtime/graph/README.md#2-run-llm-with-transformer-based-api)

```

Comment:
runtime dir had been remove from ITREX now.
pls turn to [neural-speed](https://github.com/intel/neural-speed)